cr-examples/onnx/src/main/java/oracle/code/onnx/OnnxRuntime.java

```diff
@@ -132,6 +132,10 @@ protected CachedModel computeValue(Class<?> type) {
     private static final CachedModelClassValue MODEL_CACHE = new CachedModelClassValue();
 
     public static <T> Tensor<T> execute(MethodHandles.Lookup l, OnnxFunction<Tensor<T>> codeLambda) {
+        return execute(l, codeLambda, Arena.ofAuto());
+    }
+
+    public static <T> Tensor<T> execute(MethodHandles.Lookup l, OnnxFunction<Tensor<T>> codeLambda, Arena sessionArena) {
```

Comment:
Can we make the `codeLambda` be the last parameter? Then the arguments are more easily identifiable, with the "configuration" stuff prefixed.

Reply:
Fixed here: https://github.com/openjdk/babylon/pull/334